Helper/ApiClient.php

```diff
@@ -46,7 +46,7 @@ public function prepareProductsData($order)
             $productType = $item->getData('product_type');
             if ($productType == 'simple') {
                 $_product = $productModel->load($productID);
-            } elseif ($productType == 'configurable' || $productType == 'grouped' || $productType == 'bundle') {
+            } elseif ($productType == 'configurable' || $productType == 'grouped' || $productType == 'bundle' || $productType == 'downloadable' || $productType == 'virtual') {
```

Comment:
as I understand by default Magento2 supports 6 product types, so can we replace the elseif statement (with all the conditions) with just else statement?

Reply:
yes we can let me do.